android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/utils/EnumMapper.java

```diff
@@ -0,0 +1,10 @@
+package com.appcoins.sdk.billing.utils;
+
+public class EnumMapper {
+
+  public Enum parseToEnum(Class enumClass, String string) {
```

Comment:
can we have a default value for the cases where we don't know the value?for instance if we add a new status